engine/src/main/scala/cromwell/engine/workflow/lifecycle/WorkflowInitializationActor.scala

```diff
@@ -54,10 +57,17 @@ case class WorkflowInitializationActor(workflowId: WorkflowId, workflowDescripto
   override val failureState = InitializationFailedState
   override val abortedState = InitializationsAbortedState
 
-  override val successResponse = WorkflowInitializationSucceededResponse
+  override def successResponse(data: WorkflowLifecycleActorData) = {
```

Comment:
Okee doke. Could we do it after the afternoon google sync (3:30+)?

Truth be told even when I first looked at the PR something was bugging me but I didn't know what it was and it was too low a level to bother commenting and I think it ties into the whole google doc options which I'd completely forgotten about. So while I was happy to go w/ the flow I'm 👍 on chatting about it more too